src/tests/ftest/util/command_utils.py

```diff
@@ -352,6 +352,28 @@ def stop(self):
             self._process = None
 
 
+class DaosCommand(ExecutableCommand):
+    """A class for similar daos command line tools."""
+
+    def __init__(self, namespace, command, path=""):
+        """Create DaosCommand object.
+        Specific type of command object built so command str returns:
+            <command> <options> <request> <action/subcommand> <options>
+        Args:
+            command (str): string of the command to be executed.
+            path (str): path to location of daos command binary.
+        """
+        super(DaosCommand, self).__init__(namespace, command, path)
+        self.request = BasicParameter("{}")
+        self.action = BasicParameter("{}")
+
+    def get_param_names(self):
```

Comment:
Instead of doing this, I'd suggest a combination of the previous comment's `get_params()` overwrite + using a new overridable method for getting the names to use in the `__str__()` method that, in this case, can replace `self.action` with the `self._action_command`.

Reply:
Will do.